### PR TITLE
chore: clean up some command help text

### PR DESCRIPTION
**Pull request by fibonacci1729**

### CODE REVIEW

**Comment by fibonacci1729** [comment]
closes #1254

**Review by itowlson — APPROVED** [review]

**Comment by itowlson** [comment]
+ @bacongobbler for the login change

File: src/commands/login.rs [hunk 1]
```diff
@@ -28,7 +28,7 @@ const SPIN_CLIENT_ID: &str = "583e63e9-461f-4fbe-a246-23e0fb1cad10";
 
 const DEFAULT_CLOUD_URL: &str = "https://cloud.fermyon.com/";
 
-/// Log into the Fermyon Platform.
+/// Log into the Fermyon Platform using credentials from ~/.docker/config.json.
```
Comment (bacongobbler) [inline src/commands/login.rs:31]: I don't think this makes sense at the present moment. `spin deploy` is still [using bindle API endpoints](https://github.com/fermyon/spin/blob/9cdf8aebf01f2dee6c2f1515d476f7d3642250c9/src/commands/deploy.rs#L350-L359), and when we implement OCI support, we won't be reading configuration from `~/.docker/config.json` IIRC.

Would you mind removing this until we make more progress on the OCI migration?

Comment (bacongobbler) [reply src/commands/login.rs:31]: @vdice do you have anything you want to add here?

**Review by bacongobbler — CHANGES_REQUESTED** [review]
Thanks for the ping @itowlson! Dropping my 2 cents here.

File: src/commands/login.rs (hunk 1 above)
Comment (fibonacci1729) [reply src/commands/login.rs:31]: Ahh good catch @bacongobbler . This was meant to be attached to `registry login` but i had fatally goofed.

Comment (itowlson) [reply src/commands/login.rs:31]: Oops, I somehow misread this as a remove.  Yeah we should definitely not have this text.  Thanks for the catch @bacongobbler!

@fibonacci1729 I think this used to be true of `spin registry login` (not `spin login` itself), but I'm not sure if that's still the case.

File: src/commands/registry.rs [hunk 2]
```diff
@@ -6,16 +6,13 @@ use std::{io::Read, path::PathBuf};
...
-/// Currently, the OCI commands are reusing the credentials from ~/.docker/config.json to
-/// authenticate to registries.
 #[derive(Subcommand, Debug)]
 pub enum RegistryCommands {
     /// Push a Spin application to a registry.
     Push(Push),
     /// Pull a Spin application from a registry.
     Pull(Pull),
-    /// Log in to a registry.
+    /// Log in to a registry using credentials from ~/.docker/config.json.
```
Comment (itowlson) [inline src/commands/registry.rs:15]: Looking at the `oci::auth` module I think even this no longer uses the Docker credentials, except maybe as a fallback?

```
    fn default_path() -> Result<PathBuf> {
        Ok(dirs::config_dir()
            .context("Cannot find configuration directory")?
            .join("fermyon")
            .join("registry-auth.json"))
    }
```

I'd suggest removing the auth file path altogether.

**Review by itowlson — CHANGES_REQUESTED** [review]

File: src/commands/registry.rs (hunk 2 above)
Comment (fibonacci1729) [reply src/commands/registry.rs:15]: Yeah you beat me to the words. I agree. Thanks!

**Review by bacongobbler — APPROVED** [review]

**Review by itowlson — APPROVED** [review]
Thanks and sorry for the faff!

**Pull request merged**